brain_games/game_core.py

```diff
@@ -0,0 +1,28 @@
+#!usr/bin/env python3
+
+from brain_games.games import brain_even_game
+from brain_games.cli import (
+    welcome_and_acknowledge_user,
+    print_rules,
+    ask_question,
+    ask_answer,
+    correct_answer,
+    congratulate_user,
+    game_over)
+
+
+def game_core(game):
+    user_name = welcome_and_acknowledge_user()
+    print_rules(brain_even_game.game_rules)
```

Comment:
this file should be able to work with any game, but now it uses `brain_even_game` rule even if another game will be loaded. 
```suggestion
    print_rules(game.game_rules)
```